velox/exec/tests/AggregationFuzzer.cpp

```diff
@@ -491,6 +500,31 @@ std::vector<RowVectorPtr> AggregationFuzzer::generateInputData(
   return input;
 }
 
+std::vector<RowVectorPtr> AggregationFuzzer::generateInputDataWithId(
+    std::vector<std::string> names,
+    std::vector<TypePtr> types) {
+  names.push_back("id");
+  types.push_back(BIGINT());
+  auto inputType = ROW(std::move(names), std::move(types));
+
+  std::vector<RowVectorPtr> input;
+  auto size = vectorFuzzer_.getOptions().vectorSize;
+  int64_t identifier = 0;
+  for (auto i = 0; i < FLAGS_num_batches; ++i) {
+    std::vector<VectorPtr> children;
+    for (auto i = 0; i + 1 < inputType->size(); ++i) {
+      children.push_back(vectorFuzzer_.fuzz(inputType->childAt(i), size));
+    }
+    velox::test::VectorMaker vectorMaker{pool_.get()};
+    children.push_back(vectorMaker.flatVector<int64_t>(
+        size, [&](auto /*row*/) { return identifier++; }));
+    auto augmentedInput = std::make_shared<RowVector>(
```

Comment:
It is confusing that input and augmentedInput variables are of different types (one is an std::vector, another a VectorPtr). Consider dropping augmentedInput variable altogether.

```
input.push_back(vectorMaker.rowVector(children));
```